examples/core/service/connection_main.py

```diff
@@ -0,0 +1,21 @@
+import asyncio
+
+from symphony.bdk.core.config.loader import BdkConfigLoader
+from symphony.bdk.core.symphony_bdk import SymphonyBdk
+
+
+class ConnectionMain:
+
+    @staticmethod
+    async def run():
+
+        config = BdkConfigLoader.load_from_symphony_dir("config.yaml")
+
+        async with SymphonyBdk(config) as bdk:
+            connection_service = bdk.connections()
+            user_connection = await connection_service.get_connection(13056700579873)
```

Comment:
Could we [list connections](https://developers.symphony.com/restapi/reference#list-connections) instead? So that this example is more "reusable" (as we avoid hardcoding the userId)